### ISSUE TITLE
Installation failure

### ISSUE THREAD

**Issue by matthewallen00** [hassio-addons/addon-example#118]
# Problem/Motivation

 Installation attempt from add-on store produces the following error message:

 Pulled image ghcr.io/hassio-addons/example/aarch64:4.5.0 failed on content-trust verification! 

## Expected behavior

Installation success

## Actual behavior

>  Pulled image ghcr.io/hassio-addons/example/aarch64:4.5.0 failed on content-trust verification! 

## Steps to reproduce

Click the Install button 

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by williamhelle** [comment]
Try to install it localy.
local addons are on /addons/ in the home assistant.
Remember : Security Policy :
# Security Policy

I take the security of my software and services seriously. This includes
all open source software I create, maintain or help to maintain.

If you believe you have found a security vulnerability in any repository
I maintain, including this one, please report it responsible to me as
described below.

**Please DO NOT report security vulnerabilities publicly!**

So... DO NOT create a GitHub issue for it ;)

**Comment by matthewallen00** [comment]
I've attempted to install locally following the instructions at:

https://developers.home-assistant.io/docs/add-ons/tutorial

Local addons never appear when I check for updates. I tried creating the Hello World example and it still doesn't work. Getting this example addon installed isn't crucial to running my system, maybe I'll try to see if installing from the repository gets fixed in a later update

**Comment by matthewallen00** [comment]
Turns out the issue with the local installation was with my browser cache. I had to completely wipe out the cookies and all cache and reset it and then it worked like a charm. As a follow up to the original issue of installing from the the  add-on store, due to issues I was having with some sensors I wiped the os and rebuilt from the newest release. Hoping that would clear up the issue I gave it another shot but received the same content trust verification error message. 

As performing a local install addresses the issue and I can't imagine that this add-on gets a lot of attention/high need it's ok not to waste energy fixing. This is the only add-on that I've experienced this issue with, I've installed several. Just thought I'd raise the flag that it doesn't install cleanly from the store repository.

Thanks for the replies and pointer towards local install option.
